### PR TITLE
Guidelines visibility in dark mode (Fix Issue #207)

### PR DESCRIPTION
**Pull request by 18Venus**
# Issue Title:  Visibility Issue in Dark Mode
*resolve issue:* #207 

## Type of change ☑️

What sort of change have you made:
I have fixed the CSS of the guidelines box so that the text is clearly visible in both light and dark mode.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Code style update (formatting, local variables)

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

 
## How Has This Been Tested? ⚙️

Describe how have you verified the changes made.

I have run the code and here are the screenshots of the changes: 

Light Mode(No change):
light-mode

Dark Mode(changed):
dark-mode

### CODE REVIEW

**Review by 0xabdulkhalid — CHANGES_REQUESTED** [review]
Instead of `#F4CBB2` you can use `#FFFFFF`/ `#F5F5F5`.

File: tailwind.config.js [hunk 1]
```diff
@@ -27,6 +27,12 @@ export default {
         vsm: "0px 0px 3px #a5a8a8",
         mdm: "0px 0px 5px #727575",
       },
+      backgroundColor: {
+        'dark-bg': '#2E3136', 
+      },
+      textColor: {
+        'dark-text': '#F4CBB2', 
+      },
```
Comment (0xabdulkhalid) [inline tailwind.config.js:35]: Instead of `#F4CBB2` you can use `#FFFFFF`/ `#F5F5F5`.

**Review by 0xabdulkhalid — CHANGES_REQUESTED** [review]

**Review by 0xabdulkhalid — APPROVED** [review]
LGTM 👀 

Now this PR is ready to be merged! @singodiyashubham87 🚀

**Comment by singodiyashubham87** [comment]
@18Venus Well Done, Keep contributing :)

**Pull request merged**